src/jepsen/voltdb.clj

```diff
@@ -131,36 +129,39 @@
                                  );"
                       init-schema-file "init-schema"]
                   (cu/write-file! init-schema init-schema-file)
-                  (c/exec (str "JAVA_HOME=/opt/oracle_java17 PATH=/opt/oracle_java17/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin " base-dir "/bin/voltdb")
+                  (c/exec (c/env {"JAVA_HOME" (System/getenv "JAVA_HOME")
+                                  })
+                          "bin/voltdb"
                           :init
                           :-s init-schema-file
                           :--config (str base-dir "/deployment.xml")
-                          | :tee (str base-dir "/log/stdout.log")))))
+                          ;| :tee (str base-dir "/log/stdout.log")
+                          )))
   (info node "initialized"))
 
 (defn configure!
   "Prepares config files and creates fresh DB."
   [test node]
-  (c/sudo username
           (c/cd base-dir
                 (c/upload (:license test) (str base-dir "/license.xml"))
                 (cu/write-file! (deployment-xml test) "deployment.xml")
                 (init-db! node)
-                (c/exec :ln :-f :-s (str base-dir "/voltdbroot/log/volt.log") (str base-dir "/log/volt.log")))))
+                (comment this is needed for compatibility between older versions that create a voltdbroot dir )
```

Comment:
it is better to use ";" for comments. Otherwise we might get a problem since clojure might try to interpret all arguments for "comment" function :
https://clojuredocs.org/clojure.core/comment
In any case, you are probably missing quotation mark around the comment.